ansible/roles/ci/code_style_check/license/files/gather_missing_licences.py

```diff
@@ -55,9 +55,12 @@ def get_changes_in_pr(data):
     master_branch = subprocess.run(command, text=True, stdout=subprocess.PIPE, stderr=subprocess.PIPE)
     devel_branches = ""
     for branch in master_branch.stdout.split("\n"):
-        if "devel" in branch:
+        if any(entry in branch for entry in ["devel","master","main"]):
             devel_branches = branch.strip()
-
+            break
+    if devel_branches == "":
+        print("Could not find the master branch: checks for devel, master, main")
```

Comment:
If you're adding it like this, then I'd create a variable with (example name) accepted_as_devel_branches = ["devel","master","main"] as its easier to maintain later and reuse for prints/logs.

Reply:
Thats a good point, will add.

Reply:
Thats a good point, added.